analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/PermissiveCors.CSharp11.cs

```diff
@@ -10,11 +10,23 @@ public class CorsEnabledManualAddedHeadersController : Controller
         [HttpGet]
         public void Index(string header, string headerValue)
         {
-            string part1 = "Access-Control";
-            string part2 = "Allow-Origin";
+            string accessControl = "Access-Control";
+            string allowOrigin = "Allow-Origin";
             // The Access-Control-Allow-Origin response header indicates whether the response can be shared with requesting code from the given origin.
             Response.Headers.Add("""Access-Control-Allow-Origin""", """*"""); // Noncompliant
-            Response.Headers.Add($$"""{{part1}}-{{part2}}""", "*"); // FN
+            Response.Headers.Add($$"""{{accessControl}}-{{allowOrigin}}""", "*"); // FN (at the moment we validate only constant string)
+
+            const string constAccessControl = "Access-Control";
+            const string constAllowOrigin = "Allow-Origin";
+            Response.Headers.Add($$"""{{constAccessControl}}-{{constAllowOrigin}}""", "*"); // Noncompliant
```

Comment:
Could you please also add a C#10 test case where you create a constant interpolated string and pass that as an argument? That should also work, just to document the behavior.